# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/or-tools/fix-find-scip.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Our `scip-solver` package provides a `libscip` target, not a `SCIP::libscip` target...

Upstream issue: https://github.com/google/or-tools/issues/4750

--- a/cmake/system_deps.cmake	2025-08-16 23:31:02.727246369 +0200
+++ b/cmake/system_deps.cmake	2025-08-16 23:32:02.755239712 +0200
@@ -96,6 +96,10 @@
 if(USE_SCIP)
   if(NOT BUILD_SCIP AND NOT TARGET SCIP::libscip)
     find_package(SCIP REQUIRED)
+    if(TARGET libscip AND NOT TARGET SCIP::libscip)
+      message(WARNING "SCIP::libscip not provided, tring using just libscip instead")
+      add_library(SCIP::libscip ALIAS libscip)
+    endif()
   endif()
 endif()
 
